sys/kern/subr_devmap.c

```diff
@@ -149,14 +150,27 @@ devmap_add_entry(vm_paddr_t pa, vm_size_t sz)
 	 * we end with a nice efficient section mapping.
 	 */
 	if ((pa & 0x000fffff) == 0 && (sz & 0x000fffff) == 0) {
-		akva_devmap_vaddr = trunc_1mpage(akva_devmap_vaddr - sz);
+		va = trunc_1mpage(akva_devmap_vaddr - sz);
```

Comment:
Why do you need the local variable?

Reply:
It's not really important but I left it because it felt more logically correct to have the sequence of update va / check va / writeback va.